CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Panel version check while editing the genes of a panel
 - Display unknown filter tags as "danger" marked badges
 - Open WTS variantS SNVs and SVs in new tabs
+- Variant filters redecoration from multiple classifications crashes general case report
+
```

Comment:
```suggestion
```

Reply:
Eh, we have always had two lines between the unreleased and released section? I guess we could change that if you like, but... 🤷‍♂️ Would say `prettier` have an automated opinion? 😸

Reply:
Wasn't it one? That's why I keep fixing the changelogs 😆 
Also because there is always one line in between the old releases

Reply:
I always remove it for the release branch, but keep it in all the PRs. 🤷‍♂️